### DISCUSSION TITLE
Where can I officially get the ungoogled Chromium AppImage?

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1291] [Q&A]
Hello developers,

can you show me where I can get the ungoogled Chromium AppImage version? If not available can you create a current version?

My current Linux system is Fedora 33.

**Comment by mbakke** [comment]
I don't think there is an "official" AppImage of ungoogled-chromium.

You can install Flatpak or [GNU Guix](https://guix.gnu.org/manual/en/html_node/Binary-Installation.html) to obtain a version of ungoogled-chromium that will work on Fedora.

**Comment by wchen342** [comment]
You can use `chromium-browser-privacy` from RPM Fusion, or my build on [OBS](https://software.opensuse.org//download.html?project=home%3Awchen342%3Aungoogled-chromium-fedora&package=ungoogled-chromium).
